VTEX - Pick and Pack Changes API.json

```
@@ -0,0 +1,795 @@
+                    },
+                    "itemId": {
+                        "type": "string",
+                        "description": "Unique identifier for the item."
+                    },
+                    "newQuantity": {
+                        "type": "integer",
+                        "description": "New quantity of items."
+                    },
+                    "unitMultiplier": {
+                        "type": "integer",
+                        "description": "Unit multiplier."
+                    },
+                    "reasonType": {
+                        "type": "string",
+                        "description": "Reason type."
+                    },
+                    "reasonDetail": {
+                        "type": "string",
+                        "description": "Reason detail."
+                    }
+                },
+                "example": {
+                    "type": "ADD_QUANTITY",
+                    "orderId": "1481110515526-01",
+                    "itemId": "3",
+                    "newQuantity": 4,
+                    "unitMultiplier": 1,
+                    "reasonType": "TEST",
+                    "reasonDetail": "test reason detail"
+                }
+            },
+            "RejectItemRequest": {
+                "required": [
+                    "type",
+                    "orderId",
+                    "itemId",
+                    "quantity",
+                    "reasonType",
+                    "reasonDetail"
+                ],
+                "type": "object",
+                "properties": {
+                    "type": {
+                        "type": "string",
+                        "description": "Action type (`REJECT`)."
+                    },
+                    "orderId": {
+                        "type": "string",
+                        "description": "Unique identifier of the order."
+                    },
+                    "itemId": {
+                        "type": "string",
+                        "description": "Unique identifier for the item."
+                    },
+                    "quantity": {
+                        "type": "integer",
+                        "description": "Quantity of items to reject."
+                    },
+                    "reasonType": {
+                        "type": "string",
+                        "description": "Reason type."
+                    },
+                    "reasonDetail": {
+                        "type": "string",
+                        "description": "Reason detail."
+                    }
+                },
+                "example": {
+                    "type": "REJECT",
+                    "orderId": "1481110515526-01",
+                    "itemId": "1364",
+                    "quantity": 1,
+                    "reasonType": "TEST",
+                    "reasonDetail": "test reason detail"

```

Se puder colocar reasonType e reasonDetail reais melhora muito aqui.Estou supondo que essa operação de rejeição existe para quando há quebra de estoque talvez..? Portanto talvez possamos ir com algo tipo essa sugestão abaixo. Mas nesse caso melhor validar com o time.
```suggestion
                    "reasonType": "INVENTORY",
                    "reasonDetail": "inventory unavailable"
```